### PR TITLE
Add warping to solar systems using vessel

### PR DESCRIPTION
**Pull request by MegaPiggy**

### CODE REVIEW

**Review by xen-42 — CHANGES_REQUESTED** [review]
Move `CreateVessel`, `UpdateVessel`, `SetupWarpController`, `OnReceiveWarpedBody` to their own static class please (VesselWarpHandler or something).

File: NewHorizons/Main.cs [hunk 1]
```diff
@@ -108,7 +113,25 @@ public static void ResetConfigs(bool resetTranslation = true)
...
+                    destroyStockPlanets = false,
+                    coords = new StarSystemConfig.NomaiCoordinates
+                    {
+                        x = new int[5]{ 0,3,2,1,5 },
+                        y = new int[5]{ 4,5,3,2,1 },
+                        z = new int[5]{ 4,1,2,5,0 }
+                    }
+                }
+            };
+            SystemDict["EyeOfTheUniverse"] = new NewHorizonsSystem("SolarSystem", new StarSystemConfig(), Instance)
```
Comment (xen-42) [inline NewHorizons/Main.cs:125]: Wrong name.

**Review by MegaPiggy — COMMENTED** [review]
Fixed

**Review by JohnCorby — COMMENTED** [review]
good

**Review by JohnCorby — APPROVED** [review]
good

**Review by xen-42 — APPROVED** [review]

**Review by Bwc9876 — APPROVED** [review]
Good

**Pull request merged**